### DISCUSSION TITLE
Account marked as spam

### DISCUSSION THREAD

**Discussion by Never9682** [community/community#74461] [Other Feature Feedback, Questions, & Ideas] [answered]
Good afternoon everyone, who can tell me how to solve this problem:
- I can’t do anything in github because the account is flagged....
- or rather, I can even do it, but no one sees it))

How to fix the situation?

**Comment by mw-xyz** [comment, marked as answer]
Reach out to GitHub support. They will best be able to help you. Let me know if this helps
